src/index.js

```diff
@@ -95,8 +98,17 @@ export default class NodeClient {
 
     file(file = '') {
         const base = this.base();
+        let value;
+        if (base && ABSOLUTE_URL_REGEX.test(base)) {
```

Comment:
Any reason not to use path.isAbsolute here?

Reply:
No reason. I can replace the regex.

Reply:
Just checked. isAbsolute is not doing the same thing as my test. my check is for an absolute URL, but path.isAbsolute checks for an absolute path. So `isAbsolute('/test')` is `true` but `ABSOLUTE_URL_REGEX.test('/test')` is `false`

Reply:
Ah good point.